test-integration/test_integration/test_predict.py

```diff
@@ -555,3 +555,24 @@ def test_predict_fast_build(docker_image):
     )
     assert result.returncode == 0
     assert result.stdout == "hello world\n"
+
+
+def test_predict_env_vars(docker_image, cog_binary):
+    project_dir = Path(__file__).parent / "fixtures/env-project"
+    build_process = subprocess.run(
+        [cog_binary, "build", "-t", docker_image],
+        cwd=project_dir,
+        capture_output=True,
+    )
+    assert build_process.returncode == 0
+    result = subprocess.run(
+        [cog_binary, "predict", "--debug", docker_image, "-i", "name=TEST_VAR"],
```

Comment:
Can we update the other call sites to use `cog_binary` for consistency.

Reply:
Yep! I'll submit another PR to keep this tidy.